database_containerisation/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM mysql:latest
```

Comment:
Might be best to pin this to version to ensure the ground doesn't move from under our feet